app/controllers/admin_controller.rb

```diff
@@ -102,4 +104,29 @@ def culled_games
   def about
   end
 
+  def stats_display
```

Comment:
A lot of these variables should be named with a suffix of `count`. Since we're in a dynamic language, it's incredibly important that the name convey the type of the field to prevent people from getting confused.